test/apify/crawlers/browser_crawler.test.ts

```diff
@@ -22,11 +22,25 @@ import {
     RequestList,
     sleep,
     createProxyConfiguration,
-    requestUtils,
+    requestAsBrowser,
 } from 'crawlers';
 import LocalStorageDirEmulator from '../local_storage_dir_emulator';
 import { BrowserCrawlerTest } from './basic_browser_crawler';
 
+jest.mock('@crawlers/utils/src/internals/request', () => {
```

Comment:
uh, ugly, but we will get rid of this when we use got scraping directly i hope. never really liked the partial mocking, always preferred `jest.spyOn` :]

Reply:
I actually tried using jest.spyOn, but because the import differences, it didn't apply :c (and I spent way too long debugging it only to have a 💡 moment)

Reply:
yeah i know that feeling, sometimes its more sane to add a new method just for easier mocking :] 

let's see when we use got scraping directly